_posts/2020-07-24-when-scenarios-and-reality-diverge/when-scenarios-and-reality-diverge.Rmd

```diff
@@ -0,0 +1,380 @@
+---
+title: "When Scenarios and Reality Diverge"
+author:
+  - name: Vincent Jerosch-Herold
+    url: https://github.com/vintented
+date: 07-24-2020
+output: 
+  distill::distill_article:
+    toc: true
```

Comment:
I removed `toc_float` because the output seemed insensitive to it. Maybe distill doesn't support float toc?